mod_ci/controllers.py

```diff
@@ -902,7 +908,7 @@ def start_ci():
                 if repository.get_pull(number=pr_nr).mergeable is not False:
                     add_test_entry(g.db, commit_hash, TestType.pull_request, pr_nr=pr_nr)
 
-            elif payload['action'] == 'closed':
+            elif inactive:
                 g.log.debug('PR was closed, no after hash available')
                 # Cancel running queue
                 tests = Test.query.filter(Test.pr_nr == pr_nr).all()
```

Comment:
@ArchitBhonsle We should also update the message on line 919. "PR closed", "PR marked as draft".